pkg/tfbridge/provider_test.go

```diff
@@ -5484,7 +5484,7 @@ func TestPlanResourceChangeUnknowns(t *testing.T) {
 				"maxItemsOneProp":null,
 				"setBlockProps":[],
 				"listBlockProps":[],
-				"nestedListBlockProps":[{"nestedProps":null}],
+				"nestedListBlockProps":[{"nestedProps":[]}],
```

Comment:
This is the only change in behaviour to current master related to unknowns and it seems reasonable to me - nestedProps is a list type so should not be nil.

Reply:
Yeah looks like MakeTerraformInputs substituted "04da6b54-80e4-46f7-96ec-b56ff0331ba9" into `"nestedListBlockProps":[{"nestedProps":null}],` and then the newly added pass corrected that to `[]`. This is fine.. we're already in heuristic land in this case. Good to have it covered!!